Formula/q/quint.rb

```diff
@@ -0,0 +1,56 @@
+class Quint < Formula
+  desc "Core tool for the Quint specification language"
+  homepage "https://github.com/informalsystems/quint"
+  url "https://registry.npmjs.org/@informalsystems/quint/-/quint-0.29.0.tgz"
+  sha256 "80eb1993ecb28271cfda7313304a22f866c84274082dffeec52b806cb1a6623a"
+  license "Apache-2.0"
+
+  livecheck do
+    url "https://registry.npmjs.org/@informalsystems/quint"
+    strategy :npm
+  end
+
+  depends_on "node" # engines.node >= 18
```

Comment:
```suggestion
  depends_on "node"
```
We always try to use the latest node

Reply:
Removed in:

d7848e5